/<<PKGBUILDDIR>>/workdir/CxxObject/sw/source/core/access/accdoc.o:(.data.rel.ro+0x41c): multiple definition of `typeinfo for sw::FrameFormats<sw::SpzFrameFormat*>'; /<<PKGBUILDDIR>>/workdir/CxxObject/sw/source/core/access/acccontext.o:(.data.rel.ro+0x1d8): first defined here

diff --git a/sw/inc/frameformats.hxx b/sw/inc/frameformats.hxx
index 4edb1f98b134..003f52790927 100644
--- a/sw/inc/frameformats.hxx
+++ b/sw/inc/frameformats.hxx
@@ -237,7 +237,7 @@ typedef FrameFormats<::SwTableFormat*> TableFrameFormats;
 typedef FrameFormats<sw::SpzFrameFormat*> SpzFrameFormats;
 }
 
-template class SW_DLLPUBLIC sw::FrameFormats<SwTableFormat*>;
-template class SW_DLLPUBLIC sw::FrameFormats<sw::SpzFrameFormat*>;
+// template class SW_DLLPUBLIC sw::FrameFormats<SwTableFormat*>;
+// template class SW_DLLPUBLIC sw::FrameFormats<sw::SpzFrameFormat*>;
 
 /* vim:set shiftwidth=4 softtabstop=4 expandtab: */
